src/app/child-dev-project/children/dashboard-widgets/birthday-dashboard/birthday-dashboard.component.html

```diff
@@ -0,0 +1,63 @@
+<!--
+  TODO
+  - Look for a better icon here https://fontawesome.com/v5/search
+  - Set right title, subtitle, explanation
+-->
+<app-dashboard-widget
+  icon="birthday-cake"
+  theme="child"
+  title="Birthday"
+  subtitle="Upcoming Birthdays"
+  explanation="Temporary explanation"
+>
+  <app-widget-content>
+    
+    <div *ngIf="childrendataSource.data.length > 0 && !isLoading" class="table-wrapper">
+      <table mat-table [dataSource]="childrendataSource"  >
+        <tr hidden="true">
+          <th scope="col" >Child Name </th>
```

Comment:
```suggestion
          <th scope="col">Child Name</th>
```
There is a lot of inconsistent white spaces in this file. Please clean this up so we have a general consistent styling.